lib/flame/controller/actions.rb

```diff
@@ -60,11 +59,12 @@ def inherit_actions(actions = superclass.actions, exclude: [])
 			##   end
 			def with_actions(mod, exclude: [], only: nil)
 				Module.new do
-					@mod = mod
-					@methods_to_define =
-						only || (@mod.public_instance_methods(false) - exclude)
-
-					extend ModuleWithActions
+					define_singleton_method(:included) do |target|
+						target.send(
+							:copy_actions_from, mod,
+							only || mod.public_instance_methods(false) - exclude
+						)
+					end
```

Comment:
Previously, `@mod` and `ModuleWithActions` was used for including `mod` itself. Like, if you have private and public methods in `mod` — private should remain private, public should become (public) actions. You can write tests for this.

Reply:
I've returned the old `ModuleWithActions`, which now uses `copy_actions_from`, which now is public, and in fact is updated (existed) `inherit_actions` (it looked like alias).